src/app/pages/DashboardPage/TransactionsChartCard.tsx

```diff
@@ -22,27 +22,26 @@ const TransactionsChartCardCmp: FC<TransactionsChartCardProps> = ({ chartDuratio
   const { t } = useTranslation()
   const theme = useTheme()
   const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
-
+  const statsParams = durationToQueryParams[chartDuration]
   // TODO: Replace with Emerald stats
-  const { data } = useGetConsensusStatsTps({
-    // TODO: Can interval_minutes be configured? 5 interval on 1 month is too granular
-    limit: ChartUtils.getLimitByDuration(chartDuration),
-  })
+  const { data } = useGetConsensusStatsTxVolume(statsParams)
 
-  const lineChartData = data?.data.tps_checkpoints ?? null
+  const lineChartData = data?.data.buckets.map(bucket => {
+    return {
+      bucket_start: bucket.start,
+      volume_per_second: bucket.volume / statsParams.bucket_size_seconds
+    }
+  })
 
-  const totalTransactions =
-    data?.data.tps_checkpoints?.reduce((acc, curr) => {
-      return acc + (curr?.tx_volume ?? 0)
-    }, 0) ?? 0
+  const totalTransactions = data?.data.buckets.reduce((acc, curr) => acc + curr.volume, 0) ?? 0
 
   return (
     <Card sx={{ p: 0 }}>
       <CardHeader component="h5" title={t('transactionsTpsChart.header')} sx={{ pb: 0, pl: 4, pt: 4 }} />
       <CardContent sx={{ pt: 4 }}>
-        {lineChartData !== null && (
-          <LineChart<TpsCheckpoint>
-            dataKey="tx_volume"
+        {lineChartData && (
+          <LineChart
+            dataKey="volume_per_second"
```

Comment:
Would be nice to pass in the type parameter of LineChart. For type safety on `dataKey` & `data` props.

Reply:
It's inferred and still typesafe tho